redfish/provider/resource_redfish_boot_sources_override_test.go

```diff
@@ -71,7 +71,7 @@ func testAccRedfishResourceBootSourceLegacyconfig(testingInfo TestingServerCrede
 			endpoint = "https://%s"
 			ssl_insecure = true
 		}
-	   
+	    system_id = "System.Embedded.1"
```

Comment:
check formatting

Reply:
go formatting is already done on the file, let me know if you expect some other formatting.